--- examples/11.loj	2024-06-28 17:28:56.134554150 +0300
+++ /tmp/tmp.8BCKKSHicg	2026-01-28 19:04:02.465159625 +0200
@@ -153,7 +153,7 @@
 
 non-veridical: pu'u[porpi(c0)](c1)
 non-veridical: <latmo(_)><balje'a>(c0)
-(ca'o)(c1). (so'i) x1:(je'atru(_)). selcatra(x1)
+(ca'o)(c1). (so'i)x1:(je'atru(_)). selcatra(x1)
 
 ju'a nai cy pa pu'u cy no porpi kei
 .i ju'a nai cy no ke latmo balje'a ke'e
@@ -442,7 +442,7 @@
 -----
 > mi djuno le du'u makau pu klama le zarci
 
-non-veridical: du'u[? x1. (pu)(). klama(x1,c0)](c1)
+non-veridical: du'u[?x1. (pu)(). klama(x1,c0)](c1)
 non-veridical: zarci(c0)
 djuno(mi,c1)
 
@@ -453,7 +453,7 @@
 -----
 > mi djuno le du'u la djan. kau pu klama le zarci
 
-non-veridical: du'u[? x1. (pu)(). klama(x1,c0)](c1)
+non-veridical: du'u[?x1. (pu)(). klama(x1,c0)](c1)
 non-veridical: zarci(c0)
 djuno(mi,c1)
 
@@ -464,7 +464,7 @@
 -----
 > mi djuno le du'u ma pu klama le zarci
 
-non-veridical: ? x1. du'u[(pu)(). klama(x1,c0)](c1)
+non-veridical: ?x1. du'u[(pu)(). klama(x1,c0)](c1)
 non-veridical: zarci(c0)
 djuno(mi,c1)
 
@@ -486,7 +486,7 @@
 -----
 > mi ba zgana le du'u la djan. jikau la djordj. cu zvati le panka
 
-non-veridical: du'u[? x1. zvati((x1 {??} "djordj"),c0)](c1)
+non-veridical: du'u[?x1. zvati((x1 {??} "djordj"),c0)](c1)
 non-veridical: panka(c0)
 (ba)(). zgana(mi,c1)
 
