--- examples/16.loj	2024-06-28 17:28:56.135554150 +0300
+++ /tmp/tmp.XACFiH8ivy	2026-01-28 19:04:03.888158489 +0200
@@ -179,91 +179,91 @@
 -----
 > re da zo'u da viska mi
 
-EQ(2) x1. viska(x1,mi)
+EQ(2)x1. viska(x1,mi)
 
 re da zo'u da viska mi
 
 -----
 > su'ore da zo'u da viska mi
 
-(su'o re) x1. viska(x1,mi)
+(su'o re)x1. viska(x1,mi)
 
 su'o re boi da zo'u da viska mi
 
 -----
 > re da viska mi
 
-EQ(2) x1. viska(x1,mi)
+EQ(2)x1. viska(x1,mi)
 
 re da zo'u da viska mi
 
 -----
 > su'ore da viska mi
 
-(su'o re) x1. viska(x1,mi)
+(su'o re)x1. viska(x1,mi)
 
 su'o re boi da zo'u da viska mi
 
 -----
 > re prenu cu viska mi
 
-EQ(2) x1:(prenu(_)). viska(x1,mi)
+EQ(2)x1:(prenu(_)). viska(x1,mi)
 
 re da poi ke'a prenu ku'o zo'u da viska mi
 
 -----
 > re da poi prenu cu viska mi
 
-EQ(2) x1:(prenu(_)). viska(x1,mi)
+EQ(2)x1:(prenu(_)). viska(x1,mi)
 
 re da poi ke'a prenu ku'o zo'u da viska mi
 
 -----
 > re da poi prenu zo'u da viska mi
 
-EQ(2) x1:(prenu(_)). viska(x1,mi)
+EQ(2)x1:(prenu(_)). viska(x1,mi)
 
 re da poi ke'a prenu ku'o zo'u da viska mi
 
 -----
 > ci gerku cu batci re nanmu
 
-EQ(3) x1:(gerku(_)). EQ(2) x2:(nanmu(_)). batci(x1,x2)
+EQ(3)x1:(gerku(_)). EQ(2)x2:(nanmu(_)). batci(x1,x2)
 
 ci da poi ke'a gerku ku'o re de poi ke'a nanmu ku'o zo'u da batci de
 
 -----
 > ci da poi gerku cu batci re de poi nanmu
 
-EQ(3) x1:(gerku(_)). EQ(2) x2:(nanmu(_)). batci(x1,x2)
+EQ(3)x1:(gerku(_)). EQ(2)x2:(nanmu(_)). batci(x1,x2)
 
 ci da poi ke'a gerku ku'o re de poi ke'a nanmu ku'o zo'u da batci de
 
 -----
 > ci da poi gerku ku'o re de poi nanmu zo'u da batci de
 
-EQ(3) x1:(gerku(_)). EQ(2) x2:(nanmu(_)). batci(x1,x2)
+EQ(3)x1:(gerku(_)). EQ(2)x2:(nanmu(_)). batci(x1,x2)
 
 ci da poi ke'a gerku ku'o re de poi ke'a nanmu ku'o zo'u da batci de
 
 -----
 > re de poi nanmu ku'o ci da poi gerku zo'u da batci de
 
-EQ(2) x1:(nanmu(_)). EQ(3) x2:(gerku(_)). batci(x2,x1)
+EQ(2)x1:(nanmu(_)). EQ(3)x2:(gerku(_)). batci(x2,x1)
 
 re da poi ke'a nanmu ku'o ci de poi ke'a gerku ku'o zo'u de batci da
 
 -----
 > ci gerku ce'e re nanmu cu batci
 
-EQ(3) x1:(gerku(_)). EQ(2) x2:(nanmu(_)). batci(x1,x2)
+EQ(3)x1:(gerku(_)). EQ(2)x2:(nanmu(_)). batci(x1,x2)
 
 ci da poi ke'a gerku ku'o re de poi ke'a nanmu ku'o zo'u da batci de
 
 -----
 > nu'i ci gerku re nanmu [nu'u] cu batci
 
-EQ(3) x1:(gerku(_)). EQ(2) x2:(nanmu(_)). batci(x1,x2)
+EQ(3)x1:(gerku(_)). EQ(2)x2:(nanmu(_)). batci(x1,x2)
 
 ci da poi ke'a gerku ku'o re de poi ke'a nanmu ku'o zo'u da batci de
 
@@ -272,7 +272,7 @@
 
 non-veridical: (nanmu(c1) /\ 2 mei(c1))
 non-veridical: (gerku(c0) /\ 3 mei(c0))
-FA x1:((_ =< c0)). FA x2:((_ =< c1)). batci(x1,x2)
+FA x1:((_ <= c0)). FA x2:((_ <= c1)). batci(x1,x2)
 
 ju'a nai ge cy pa nanmu gi cy pa re mei
 .i ju'a nai ge cy no gerku gi cy no ci mei
@@ -432,14 +432,14 @@
 -----
 > noda rode zo'u da prami de
 
-EQ(0) x1. FA x2. prami(x1,x2)
+EQ(0)x1. FA x2. prami(x1,x2)
 
 no da ro de zo'u da prami de
 
 -----
 > naku noda rode zo'u da prami de
 
-!EQ(0) x1. FA x2. prami(x1,x2)
+!EQ(0)x1. FA x2. prami(x1,x2)
 
 na ku no da ro de zo'u da prami de
 
@@ -503,8 +503,8 @@
 > Some children do not go to school.
 
 Morphology error:
-	{Some children do not go to school. }
-	      ^
+	{  Some children do not go to school.}
+	        ^
 
 -----
 > su'oda poi verba na klama su'ode poi ckule
@@ -640,7 +640,7 @@
 > la djein. le zarci ganai dzukla ginai bajrykla
 
 non-veridical: zarci(c0)
-(bajrykla("djein",c0) --> !dzukla("djein",c0))
+(bajrykla("djein",c0) -> !dzukla("djein",c0))
 
 ju'a nai cy no zarci
 .i ga nai la djein. bajrykla cy no gi na ku la djein. dzukla cy no
@@ -681,7 +681,7 @@
 
 non-veridical: zarci(c1)
 non-veridical: zarci(c0)
-(bajrykla("djein",c1) --> !dzukla("djein",c0))
+(bajrykla("djein",c1) -> !dzukla("djein",c0))
 
 ju'a nai cy pa zarci
 .i ju'a nai cy no zarci
@@ -725,14 +725,14 @@
 -----
 > ci da poi mlatu cu blabi .ije re da cu barda
 
-(EQ(3) x1:(mlatu(_)). blabi(x1) /\ EQ(2) x1. barda(x1))
+(EQ(3)x1:(mlatu(_)). blabi(x1) /\ EQ(2)x1. barda(x1))
 
 ge ci da poi ke'a mlatu ku'o zo'u da blabi gi re da zo'u da barda
 
 -----
 > ci da poi prenu cu se ralju pa da
 
-EQ(3) x1:(prenu(_)). EQ(1) x2. ralju(x2,x1)
+EQ(3)x1:(prenu(_)). EQ(1)x2. ralju(x2,x1)
 
 ci da poi ke'a prenu ku'o pa de zo'u de ralju da
 
