--- examples/17.loj	2024-06-28 17:28:56.135554150 +0300
+++ /tmp/tmp.al1o1UEbP7	2026-01-28 19:04:04.012158390 +0200
@@ -208,7 +208,7 @@
 -----
 > dei vasru vo lerfu po'u me'o .ebu
 
-EQ(4) x1:((lerfu(_) /\ (_ == [MEX: MexLerfuString [LerfuChar 'e']]))). vasru(dei,x1)
+EQ(4)x1:((lerfu(_) /\ (_ == [MEX: MexLerfuString [LerfuChar 'e']]))). vasru(dei,x1)
 
 vo da poi ge ke'a lerfu gi ke'a du me'o MexLerfuString [LerfuChar 'e'] lo'o ku'o zo'u dei vasru da
 
@@ -326,7 +326,7 @@
 -----
 > la me dy ny. .abu
 
-[Fragment: [Name: (_ =< dna) ]]
+[Fragment: [Name: (_ <= dna) ]]
 
 la me dy ny abu me'u ku
 
