--- examples/18.loj	2024-06-28 17:28:56.135554150 +0300
+++ /tmp/tmp.xQdNcuCFV9	2026-01-28 19:04:04.437158051 +0200
@@ -277,7 +277,7 @@
 -----
 > mi catlu pa prenu
 
-EQ(1) x1:(prenu(_)). catlu(mi,x1)
+EQ(1)x1:(prenu(_)). catlu(mi,x1)
 
 pa da poi ke'a prenu ku'o zo'u mi catlu da
 
@@ -291,35 +291,35 @@
 -----
 > mi catlu so'a prenu
 
-(so'a) x1:(prenu(_)). catlu(mi,x1)
+(so'a)x1:(prenu(_)). catlu(mi,x1)
 
 so'a boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu so'e prenu
 
-(so'e) x1:(prenu(_)). catlu(mi,x1)
+(so'e)x1:(prenu(_)). catlu(mi,x1)
 
 so'e boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu so'i prenu
 
-(so'i) x1:(prenu(_)). catlu(mi,x1)
+(so'i)x1:(prenu(_)). catlu(mi,x1)
 
 so'i boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu so'o prenu
 
-(so'o) x1:(prenu(_)). catlu(mi,x1)
+(so'o)x1:(prenu(_)). catlu(mi,x1)
 
 so'o boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu so'u prenu
 
-(so'u) x1:(prenu(_)). catlu(mi,x1)
+(so'u)x1:(prenu(_)). catlu(mi,x1)
 
 so'u boi da poi ke'a prenu ku'o zo'u mi catlu da
 
@@ -328,7 +328,7 @@
 
 gunma(c1,c0)
 non-veridical: nanba(c0)
-(pi ro) x1:((_ =< c1)). citka(mi,x1)
+(pi ro)x1:((_ <= c1)). citka(mi,x1)
 
 cy pa gunma cy no
 .i ju'a nai cy no nanba
@@ -337,7 +337,7 @@
 -----
 > mi catlu no'o prenu
 
-(no'o) x1:(prenu(_)). catlu(mi,x1)
+(no'o)x1:(prenu(_)). catlu(mi,x1)
 
 no'o boi da poi ke'a prenu ku'o zo'u mi catlu da
 
@@ -346,7 +346,7 @@
 
 gunma(c1,c0)
 non-veridical: nanba(c0)
-(pi no'o) x1:((_ =< c1)). citka(mi,x1)
+(pi no'o)x1:((_ <= c1)). citka(mi,x1)
 
 cy pa gunma cy no
 .i ju'a nai cy no nanba
@@ -355,21 +355,21 @@
 -----
 > mi catlu da'a re prenu
 
-(da'a re) x1:(prenu(_)). catlu(mi,x1)
+(da'a re)x1:(prenu(_)). catlu(mi,x1)
 
 da'a re boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu da'a so'u prenu
 
-(da'a so'u) x1:(prenu(_)). catlu(mi,x1)
+(da'a so'u)x1:(prenu(_)). catlu(mi,x1)
 
 da'a so'u boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > ro ratcu ka'e citka da'a ratcu
 
-FA x1:(ratcu(_)). (ka'e)(). (da'a) x2:(ratcu(_)). citka(x1,x2)
+FA x1:(ratcu(_)). (ka'e)(). (da'a)x2:(ratcu(_)). citka(x1,x2)
 
 ro da poi ke'a ratcu ku'o ka'e ku da'a boi de poi ke'a ratcu ku'o zo'u da citka de
 
@@ -390,14 +390,14 @@
 -----
 > mi ponse ma'u rupnu
 
-(ma'u) x1:(rupnu(_)). ponse(mi,x1)
+(ma'u)x1:(rupnu(_)). ponse(mi,x1)
 
 ma'u boi da poi ke'a rupnu ku'o zo'u mi ponse da
 
 -----
 > mi ponse rau rupnu
 
-(rau) x1:(rupnu(_)). ponse(mi,x1)
+(rau)x1:(rupnu(_)). ponse(mi,x1)
 
 rau boi da poi ke'a rupnu ku'o zo'u mi ponse da
 
@@ -413,7 +413,7 @@
 -----
 > mi speni so'ici prenu
 
-(so'i ci) x1:(prenu(_)). speni(mi,x1)
+(so'i ci)x1:(prenu(_)). speni(mi,x1)
 
 so'i ci boi da poi ke'a prenu ku'o zo'u mi speni da
 
@@ -455,28 +455,28 @@
 -----
 > mi catlu su'e re prenu
 
-(su'e re) x1:(prenu(_)). catlu(mi,x1)
+(su'e re)x1:(prenu(_)). catlu(mi,x1)
 
 su'e re boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu su'o re prenu
 
-(su'o re) x1:(prenu(_)). catlu(mi,x1)
+(su'o re)x1:(prenu(_)). catlu(mi,x1)
 
 su'o re boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu me'i re prenu
 
-(me'i re) x1:(prenu(_)). catlu(mi,x1)
+(me'i re)x1:(prenu(_)). catlu(mi,x1)
 
 me'i re boi da poi ke'a prenu ku'o zo'u mi catlu da
 
 -----
 > mi catlu za'u re prenu
 
-(za'u re) x1:(prenu(_)). catlu(mi,x1)
+(za'u re)x1:(prenu(_)). catlu(mi,x1)
 
 za'u re boi da poi ke'a prenu ku'o zo'u mi catlu da
 
@@ -578,7 +578,7 @@
 -----
 > mi poi pamei cu cusku dei
 
-(1 mei(c0) /\ (c0 =< mi))
+(1 mei(c0) /\ (c0 <= mi))
 cusku(c0,dei)
 
 ge cy no pa mei gi cy no me mi me'u
@@ -863,7 +863,7 @@
 > vei ci .a vo ve'o prenu cu klama le zarci
 
 non-veridical: zarci(c0)
-(EQ(3) x1:(prenu(_)). klama(x1,c0) \/ EQ(4) x1:(prenu(_)). klama(x1,c0))
+(EQ(3)x1:(prenu(_)). klama(x1,c0) \/ EQ(4)x1:(prenu(_)). klama(x1,c0))
 
 ju'a nai cy no zarci
 .i ga ci da poi ke'a prenu ku'o zo'u da klama cy no gi vo da poi ke'a prenu ku'o zo'u da klama cy no
@@ -872,7 +872,7 @@
 > vei ga ci gi vo ve'o prenu cu klama le zarci
 
 non-veridical: zarci(c0)
-(EQ(3) x1:(prenu(_)). klama(x1,c0) \/ EQ(4) x1:(prenu(_)). klama(x1,c0))
+(EQ(3)x1:(prenu(_)). klama(x1,c0) \/ EQ(4)x1:(prenu(_)). klama(x1,c0))
 
 ju'a nai cy no zarci
 .i ga ci da poi ke'a prenu ku'o zo'u da klama cy no gi vo da poi ke'a prenu ku'o zo'u da klama cy no
@@ -960,7 +960,7 @@
 > mi viska vei mo'e lo'e lanzu ve'o cinfo
 
 lanzu(c0)
-[c0] x1:(cinfo(_)). viska(mi,x1)
+[c0]x1:(cinfo(_)). viska(mi,x1)
 
 cy no lanzu
 .i mo'e cy no te'u da poi ke'a cinfo ku'o zo'u mi viska da
@@ -989,9 +989,9 @@
 -----
 > li re na'u mo re du li vo
 
-? R1. ([R1(_)](2,2) == 4)
+?R1. ([R1(_)](2,2) == 4)
 
-mo cei broda zo'u li pe'o na'u broda te'u re boi re boi ku'e lo'o du li vo boi lo'o
+tu'o mo cei broda zo'u li pe'o na'u broda te'u re boi re boi ku'e lo'o du li vo boi lo'o
 
 -----
 > nu'a su'i
@@ -1053,7 +1053,7 @@
 -----
 > la zel. poi gunta la tebes. pu nanmu
 
-((pu)(). gunta(c0,"tebes") /\ (c0 =< "zel"))
+((pu)(). gunta(c0,"tebes") /\ (c0 <= "zel"))
 nanmu(c0)
 
 ge pu ku zo'u cy no gunta la tebes. gi cy no me la zel. me'u
