--- examples/19.loj	2024-06-28 17:28:56.135554150 +0300
+++ /tmp/tmp.GeuCNaPgnf	2026-01-28 19:04:04.746157804 +0200
@@ -69,7 +69,7 @@
 
 gunma(c1,c0)
 jdini(c0)
-(ponse(do) --> djica(do))
+(ponse(do) -> djica(do))
 
 cy pa gunma cy no
 .i cy no jdini
@@ -80,7 +80,7 @@
 
 gunma(c1,c0)
 jdini(c0)
-(ponse(do,c1) --> djica(do,c1))
+(ponse(do,c1) -> djica(do,c1))
 
 cy pa gunma cy no
 .i cy no jdini
@@ -91,7 +91,7 @@
 
 gunma(c1,c0)
 jdini(c0)
-(ponse(do,c1) --> djica(do,{tu'a}(c1)))
+(ponse(do,c1) -> djica(do,{tu'a}(c1)))
 
 cy pa gunma cy no
 .i cy no jdini
@@ -110,7 +110,7 @@
 > ma klama le zarci
 
 non-veridical: zarci(c0)
-? x1. klama(x1,c0)
+?x1. klama(x1,c0)
 
 ju'a nai cy no zarci
 .i ma goi ko'a zo'u ko'a klama cy no
@@ -118,7 +118,7 @@
 -----
 > do klama ma
 
-? x1. klama(do,x1)
+?x1. klama(do,x1)
 
 ma goi ko'a zo'u do klama ko'a
 
@@ -134,7 +134,7 @@
 -----
 > ma klama ma
 
-? x1. ? x2. klama(x1,x2)
+?x1. ?x2. klama(x1,x2)
 
 ma goi ko'a ma goi ko'e zo'u ko'a klama ko'e
 
@@ -150,7 +150,7 @@
 -----
 > ma fa'u ma klama ma fa'u ma
 
-? x1. ? x2. ? x3. ? x4. klama((x1 {fa'u} x2),(x3 {fa'u} x4))
+?x1. ?x2. ?x3. ?x4. klama((x1 {fa'u} x2),(x3 {fa'u} x4))
 
 ma goi ko'a ma goi ko'e ma goi ko'i ma goi ko'o zo'u ko'a fa'u ke ko'e ke'e klama ko'i fa'u ke ko'o ke'e
 
@@ -168,14 +168,14 @@
 -----
 > la lojban. mo
 
-? R1. R1("lojban")
+?R1. R1("lojban")
 
-mo cei broda zo'u la lojban. broda
+tu'o mo cei broda zo'u la lojban. broda
 
 -----
 > do viska xo prenu
 
-(xo) x1:(prenu(_)). viska(do,x1)
+(xo)x1:(prenu(_)). viska(do,x1)
 
 xo boi da poi ke'a prenu ku'o zo'u do viska da
 
@@ -433,8 +433,8 @@
 > zoi gy. John is a man .gy. cu glico jufra
 
 Morphology error:
-	{zoi gy. John is a man .gy. cu glico jufra }
-	         ^
+	{  zoi gy. John is a man .gy. cu glico jufra}
+	           ^
 
 -----
 > mi djuno fi le valsi po'u zoi gy. gyrations .gy.
@@ -456,8 +456,8 @@
 > zoi ry. sku .ry. cu rafsi zo cusku
 
 Morphology error:
-	{zoi ry. sku .ry. cu rafsi zo cusku }
-	         ^
+	{  zoi ry. sku .ry. cu rafsi zo cusku}
+	           ^
 
 -----
 > zo .bab. cmene la bab.
@@ -491,8 +491,8 @@
 > la'o dy. Goethe .dy. cu me la'o ly. Homo sapiens .ly.
 
 Morphology error:
-	{la'o dy. Goethe .dy. cu me la'o ly. Homo sapiens .ly. }
-	          ^
+	{  la'o dy. Goethe .dy. cu me la'o ly. Homo sapiens .ly.}
+	            ^
 
 -----
 > mi viska la ba'e .djordj.
