--- examples/3.loj	2024-06-28 17:28:56.136554151 +0300
+++ /tmp/tmp.ks3wwNtFQI	2026-01-28 19:03:59.930161650 +0200
@@ -8,15 +8,15 @@
 > .i,ai,i,ai,on.
 
 Morphology error:
-	{.i,ai,i,ai,on. }
-	  ^
+	{  .i,ai,i,ai,on.}
+	    ^
 
 -----
 > .i,iai,ii,iai,ion.
 
 Morphology error:
-	{.i,iai,ii,iai,ion. }
-	  ^
+	{  .i,iai,ii,iai,ion.}
+	    ^
 
 -----
 > .i'ai'i'ai'on.
@@ -86,15 +86,11 @@
 
 [Fragment: ]
 
-
-
 -----
 > pu,je,nai,ca,je,ba
 
 [Fragment: ]
 
-
-
 -----
 > ninmu
 
