--- examples/6.loj	2024-06-28 17:28:56.136554151 +0300
+++ /tmp/tmp.870KuSPE9j	2026-01-28 19:04:00.909160868 +0200
@@ -305,7 +305,7 @@
 > re do cadzu le bisli
 
 non-veridical: bisli(c0)
-EQ(2) x1:((_ =< do)). cadzu(x1,c0)
+EQ(2)x1:((_ <= do)). cadzu(x1,c0)
 
 ju'a nai cy no bisli
 .i re da poi ke'a me do me'u ku'o zo'u da cadzu cy no
@@ -313,7 +313,7 @@
 -----
 > mi ponse su'o ci cutci
 
-(su'o ci) x1:(cutci(_)). ponse(mi,x1)
+(su'o ci)x1:(cutci(_)). ponse(mi,x1)
 
 su'o ci boi da poi ke'a cutci ku'o zo'u mi ponse da
 
@@ -321,7 +321,7 @@
 > ro do cadzu le bisli
 
 non-veridical: bisli(c0)
-FA x1:((_ =< do)). cadzu(x1,c0)
+FA x1:((_ <= do)). cadzu(x1,c0)
 
 ju'a nai cy no bisli
 .i ro da poi ke'a me do me'u ku'o zo'u da cadzu cy no
@@ -338,7 +338,7 @@
 -----
 > mi cusku ro lu do cadzu le bisli li'u
 
-FA x1:((_ =< << non-veridical: bisli(c0)
+FA x1:((_ <= << non-veridical: bisli(c0)
 cadzu(do,c0) >>)). cusku(mi,x1)
 
 ro da poi ke'a me lu ju'a nai cy no bisli
@@ -347,7 +347,7 @@
 -----
 > mi cusku su'o lu do cadzu le bisli li'u
 
-EX x1:((_ =< << non-veridical: bisli(c0)
+EX x1:((_ <= << non-veridical: bisli(c0)
 cadzu(do,c0) >>)). cusku(mi,x1)
 
 su'o da poi ke'a me lu ju'a nai cy no bisli
@@ -356,7 +356,7 @@
 -----
 > mi cusku re lu do cadzu le bisli li'u
 
-EQ(2) x1:((_ =< << non-veridical: bisli(c0)
+EQ(2)x1:((_ <= << non-veridical: bisli(c0)
 cadzu(do,c0) >>)). cusku(mi,x1)
 
 re da poi ke'a me lu ju'a nai cy no bisli
@@ -366,7 +366,7 @@
 > re le gerku cu blabi
 
 non-veridical: gerku(c0)
-EQ(2) x1:((_ =< c0)). blabi(x1)
+EQ(2)x1:((_ <= c0)). blabi(x1)
 
 ju'a nai cy no gerku
 .i re da poi ke'a me cy no me'u ku'o zo'u da blabi
@@ -375,7 +375,7 @@
 > re le ci gerku cu blabi
 
 non-veridical: (gerku(c0) /\ 3 mei(c0))
-EQ(2) x1:((_ =< c0)). blabi(x1)
+EQ(2)x1:((_ <= c0)). blabi(x1)
 
 ju'a nai ge cy no gerku gi cy no ci mei
 .i re da poi ke'a me cy no me'u ku'o zo'u da blabi
@@ -393,7 +393,7 @@
 > [ro] le ci gerku cu blabi
 
 non-veridical: (gerku(c0) /\ 3 mei(c0))
-FA x1:((_ =< c0)). blabi(x1)
+FA x1:((_ <= c0)). blabi(x1)
 
 ju'a nai ge cy no gerku gi cy no ci mei
 .i ro da poi ke'a me cy no me'u ku'o zo'u da blabi
@@ -402,7 +402,7 @@
 > ci lo [ro] gerku cu blabi
 
 (gerku(c0) /\ (ro) mei(c0))
-EQ(3) x1:((_ =< c0)). blabi(x1)
+EQ(3)x1:((_ <= c0)). blabi(x1)
 
 ge cy no gerku gi cy no ro mei
 .i ci da poi ke'a me cy no me'u ku'o zo'u da blabi
@@ -411,7 +411,7 @@
 > [so'o] lo ci gerku cu blabi
 
 (gerku(c0) /\ 3 mei(c0))
-(so'o) x1:((_ =< c0)). blabi(x1)
+(so'o)x1:((_ <= c0)). blabi(x1)
 
 ge cy no gerku gi cy no ci mei
 .i so'o boi da poi ke'a me cy no me'u ku'o zo'u da blabi
@@ -419,7 +419,7 @@
 -----
 > ci gerku [ku] cu blabi
 
-EQ(3) x1:(gerku(_)). blabi(x1)
+EQ(3)x1:(gerku(_)). blabi(x1)
 
 ci da poi ke'a gerku ku'o zo'u da blabi
 
@@ -427,7 +427,7 @@
 > mi ponse su'o ci lo cutci
 
 cutci(c0)
-(su'o ci) x1:((_ =< c0)). ponse(mi,x1)
+(su'o ci)x1:((_ <= c0)). ponse(mi,x1)
 
 cy no cutci
 .i su'o ci boi da poi ke'a me cy no me'u ku'o zo'u mi ponse da
@@ -435,14 +435,14 @@
 -----
 > re do cu nanmu
 
-EQ(2) x1:((_ =< do)). nanmu(x1)
+EQ(2)x1:((_ <= do)). nanmu(x1)
 
 re da poi ke'a me do me'u ku'o zo'u da nanmu
 
 -----
 > le re do cu nanmu
 
-non-veridical: ((c0 =< do) /\ 2 mei(c0))
+non-veridical: ((c0 <= do) /\ 2 mei(c0))
 nanmu(c0)
 
 ju'a nai ge cy no me do me'u gi cy no re mei
@@ -452,7 +452,7 @@
 > re le ci cribe cu bunre
 
 non-veridical: (cribe(c0) /\ 3 mei(c0))
-EQ(2) x1:((_ =< c0)). bunre(x1)
+EQ(2)x1:((_ <= c0)). bunre(x1)
 
 ju'a nai ge cy no cribe gi cy no ci mei
 .i re da poi ke'a me cy no me'u ku'o zo'u da bunre
@@ -460,7 +460,7 @@
 -----
 > le re le ci cribe cu bunre
 
-non-veridical: ((c1 =< c0) /\ 2 mei(c1))
+non-veridical: ((c1 <= c0) /\ 2 mei(c1))
 non-veridical: (cribe(c0) /\ 3 mei(c0))
 bunre(c1)
 
@@ -471,9 +471,9 @@
 -----
 > pa le re le ci cribe cu bunre
 
-non-veridical: ((c1 =< c0) /\ 2 mei(c1))
+non-veridical: ((c1 <= c0) /\ 2 mei(c1))
 non-veridical: (cribe(c0) /\ 3 mei(c0))
-EQ(1) x1:((_ =< c1)). bunre(x1)
+EQ(1)x1:((_ <= c1)). bunre(x1)
 
 ju'a nai ge cy pa me cy no me'u gi cy pa re mei
 .i ju'a nai ge cy no cribe gi cy no ci mei
@@ -701,7 +701,7 @@
 -----
 > ro da poi prenu cu prami pa de poi finpe
 
-FA x1:(prenu(_)). EQ(1) x2:(finpe(_)). prami(x1,x2)
+FA x1:(prenu(_)). EQ(1)x2:(finpe(_)). prami(x1,x2)
 
 ro da poi ke'a prenu ku'o pa de poi ke'a finpe ku'o zo'u da prami de
 
@@ -747,7 +747,7 @@
 -----
 > mi viska le mlatu ku poi zo'e zbasu ke'a loi slasi
 
-(zbasu(c1,c4,c3) /\ (c4 =< c0))
+(zbasu(c1,c4,c3) /\ (c4 <= c0))
 gunma(c3,c2)
 slasi(c2)
 non-veridical: mlatu(c0)
@@ -762,7 +762,7 @@
 -----
 > do klama ma
 
-? x1. klama(do,x1)
+?x1. klama(do,x1)
 
 ma goi ko'a zo'u do klama ko'a
 
@@ -791,8 +791,8 @@
 > mi cusku zoi kuot. I'm John .kuot
 
 Morphology error:
-	{mi cusku zoi kuot. I'm John .kuot }
-	                    ^
+	{  mi cusku zoi kuot. I'm John .kuot}
+	                      ^
 
 -----
 > li vo
